src/System.Net.Primitives/src/System/Net/IWebProxy.cs

```diff
@@ -1,18 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Ugh, didn't mean to commit that... this keeps getting automatically deleted in my local repository for some reason...